### PR TITLE
M3 4418 e2e around linode backups

### PR DESCRIPTION
**Pull request by ndukatz**
3 tests around backups: enable backups, snapshot blocked while linode busy, creating a snapshot. I would like to make a test to create a linode from a backup in the future. Still working on figuring out how to do that.

I am not 100% sure that waiting for `Provisioning` & `Booting` to be gone is the best way to handle making sure that we can actually create the snapshot on the 3rd test since it takes a little longer, but it does seem to work well. 

**How to verify that that the test works:**
- yarn up
- yarn cy:debug
- Choose test backup-linode.spec.ts
- Test(s) should pass

**Extra check (runs headless):**
- yarn cy:e2e -s ./cypress/integration/linodes/backup-linode.spec.ts
- Result should be "✔  All specs passed!"

Make sure your .env contains:
REACT_APP_LOGIN_ROOT='https://login.linode.com'
REACT_APP_API_ROOT='https://api.linode.com/v4'
REACT_APP_APP_ROOT='http://localhost:3000'
REACT_APP_LAUNCH_DARKLY_ID='*************************' (CMR flag)
REACT_APP_CLIENT_ID='********************' (https://cloud.linode.com/profile/clients, copy ID)
MANAGER_OAUTH='**************************************************************************' 
(create in https://cloud.linode.com/profile/tokens > "Add a Person Access Token", Save the token that appears in the modal, you will only see it once)

### CODE REVIEW

**Review by johnwcallahan — COMMENTED** [review]
The first test wasn't passing for me at first (failing at asserting "[Enable Backups](https://github.com/linode/manager/pull/6868/files#diff-b0376f43bf481c080d70212e1ef05bc4R21)" is visible). That's because the account I was testing from was Managed, and had automatic backups enabled by default.

I'm not sure how to best handle this situation, since from a Managed account you _can't_ create a Linode without backups enabled. 

We should at least handle it so that the test doesn't fail if you have automatic backups enabled. Maybe eventually we have two runners, one with and without Managed.

Besides that issue, tests are working for me.

**Review by johnwcallahan — APPROVED** [review]

**Comment by acourdavault** [comment]
Same issue than John, my account is managed.
I imagine we could mock it

File: packages/manager/cypress/integration/linodes/backup-linode.spec.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  createLinode,
+  createLinodeWithBackupsEnabled,
+  deleteLinodeById
+} from '../../support/api/linodes';
+import { waitForAppLoad } from '../../support/ui/common';
+import { assertToast } from '../../support/ui/events';
+
+describe('linode backups', () => {
+  it('enable backups', () => {
+    cy.visitWithLogin('/dashboard');
```
Comment (acourdavault) [inline packages/manager/cypress/integration/linodes/backup-linode.spec.ts:12]: we probably can remove this

File: packages/manager/cypress/integration/linodes/backup-linode.spec.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    createLinode().then(linode => {
+      cy.server();
+      cy.route({
+        method: 'POST',
+        url: `*/linode/instances/${linode.id}/backups/enable`
+      }).as('enableBackups');
+      waitForAppLoad();
```
Comment (acourdavault) [inline packages/manager/cypress/integration/linodes/backup-linode.spec.ts:19]: here we should remove this and add a route on `url: '*/account/settings*'` mocking the response to not be in q managed case.
Then wait for that request

**Review by acourdavault — APPROVED** [review]
So the test is good as it adds coverage, we could in a separate PR handle the mocking of the "managed" or "backups enabled" flags in the account to be able to run this test on a managed account

**Comment by ndukatz** [comment]
Ended up just having it skip through some parts if the account is managed for now. I couldn't get the mock to work

**Pull request merged**